src/ComplexTypes/TLCType.ts

```diff
@@ -0,0 +1,26 @@
+import { AttributeGroupItem } from "../Interfaces/AttributeGroupItem";
+import { HasChildrenMap } from "../Interfaces/HasChildrenMap";
+import { NodeRules } from "../Interfaces/NodeRules";
+import { EIdAttribute } from "../Attributes/EIdAttribute";
+import { ShowAsAttribute } from "../Attributes/ShowAsAttribute";
+import { HrefAttribute } from "../Attributes/HrefAttribute";
+import { ShortFormAttribute } from "../Attributes/ShortFormAttribute";
+import { GUIDAttribute } from "../Attributes/GUIDAttribute";
+import { WIdAttribute } from "../Attributes/WIdAttribute";
+
+/**
+ * The complex type tlctype is used by top level classes i.e.
+ * TLCConcept that are used in the references node
+ */
+export class TLCType implements HasChildrenMap {
```

Comment:
Is this a valid complex type? I cannot locate it. Please share source

Reply:
Sorry, I don't seem to understand complex types. I added it under complex types because I needed to define the ATTRIBUTE_GROUPS. Is [this](https://github.com/musale/nkyimu/blob/update/add-references-node/src/ComplexTypes/TLCType.ts)  helpful?

Reply:
Let's have a chat on this